lib/Controller/User.php

```diff
@@ -748,6 +956,49 @@ public function changePassword()
             'passwordUpdate' => true
         ]);
 
+        $user->isPasswordChangeRequired = 0;
+        $user->save();
+
+        // Return
+        $this->getState()->hydrate([
+            'message' => __('Password Changed'),
+            'id' => $user->userId,
+            'data' => $user
+        ]);
+    }
+
+    /**
+     * Force User Password Change
+     */
+    public function forceChangePasswordPage()
+    {
+        $this->getState()->template = 'user-force-change-password-page';
```

Comment:
we need something in here that redirects the user to the homepage if their change password flag isn't set to 1.